core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/MetadataImpl.java

```diff
@@ -152,7 +154,19 @@ public boolean isEmpty() {
         if (StringUtils.isBlank(getPropertyName())) {
             return true;
         } else {
-            return combinedProperties.get(getPropertyName()) == null;
+            Object val = combinedProperties.get(getPropertyName());
+            if (val == null) {
+                return true;
+            } else if (val instanceof String) {
+                return StringUtils.isBlank((String) val);
+            } else if (val instanceof Object[]) {
+                return ArrayUtils.isEmpty((Object[]) val);
```

Comment:
I actually had a use case for a client where there was a property that was a String[] but the first value was empty.  Ideally we can still handle that case as well.